src/ControlSystem/Actions/Initialization.hpp

```diff
@@ -32,9 +32,9 @@ namespace Actions {
  *   - `control_system::Tags::Averager<2>`
  *   - `control_system::Tags::Controller<2>`
  *   - `control_system::Tags::TimescaleTuner`
+ *   - `control_system::Tags::ControlSystemName`
```

Comment:
Doing `ControlSystem::name()` everywhere would require that any action you want to run on the ControlComponent be templated on the control system (in case you need the name). That's a fairly restrictive requirement. Adding a tag for the name in the box relaxes this requirement because as long as you have access to the box (which is more readily passed around) you have access to the name.

Reply:
Ok I think passing around the control system template might be the way to go, but let's keep things as they are for now.

Reply:
Other possibilities to consider later: get the control system from the parallel component or put the control system struct itself in the DataBox.